gp-includes/router.php

```diff
@@ -181,6 +181,10 @@ public function route() {
 
 		$url_path = gp_url_path( gp_url_public_root() );
 
+		// Check if the last character of the text is not a slash, and add it if needed
+		if ( substr( $_SERVER['REQUEST_URI'], -1 ) !== '/' ) {
+			$_SERVER['REQUEST_URI'] .= '/';
```

Comment:
If we need to modify `$_SERVER['REQUEST_URI']` then we'd use `trailingslashit()`